elytron-oidc-client/src/test/java/org/wildfly/extension/elytron/oidc/ExpressionsTestCase.java

```diff
@@ -15,34 +15,48 @@
 import org.jboss.as.subsystem.test.AbstractSubsystemTest;
 import org.jboss.as.subsystem.test.AdditionalInitialization;
 import org.jboss.as.subsystem.test.KernelServices;
+import org.jboss.as.version.Stability;
 import org.junit.Assert;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
 
 /**
  * Subsystem parsing test case.
  *
  * <a href="mailto:araskar@redhat.com">Ashpan Raskar</a>
  */
+@RunWith(Parameterized.class)
 public class ExpressionsTestCase extends AbstractSubsystemTest {
 
     private KernelServices services = null;
 
-    public ExpressionsTestCase() {
+    @Parameterized.Parameters
+    public static Iterable<ElytronOidcSubsystemSchema> parameters() {
+        return ElytronOidcSubsystemSchema.CURRENT.values();
+    }
+    public ExpressionsTestCase(ElytronOidcSubsystemSchema schema) {
```

Comment:
Please note that while I don't use schema in this function, I can't drop it because otherwise I get "IllegalArgument wrong number of arguments". 
I think it is due to the parametrized nature of the tests.